device-types/Nokia/7210-SAS-Mxp-22F2C-4SFPP.yaml

```diff
@@ -0,0 +1,75 @@
+---
+manufacturer: Nokia
+model: 7210 SAS-Mxp 22F2C 4SFPP
+slug: nokia-7210-sas-mxp-22f2c-4sfpp
+part_number: 3HE10075ABRC01
+u_height: 2
+is_full_depth: false
+subdevice_role: parent
```

Comment:
`subdevice_role: parent` is only to be used if this device is either a parent (i.e. a chassis device that houses other devices), or or a child (i.e. this device slots into a chassis device). Generally parent devices have `device-bays` defined for their children devices.

**If this is not actually a parent device, please remove this definition**

Reply:
Removed